yarn.lock

```diff
@@ -2254,10 +2232,15 @@ lodash.templatesettings@^4.0.0:
   dependencies:
     lodash._reinterpolate "~3.0.0"
 
-lodash@^4.0.0, lodash@^4.1.0, lodash@^4.15.0, lodash@^4.17.4, lodash@^4.2.0, lodash@^4.2.1, lodash@^4.3.0:
+lodash@^4.0.0, lodash@^4.1.0, lodash@^4.15.0, lodash@^4.2.0, lodash@^4.2.1, lodash@^4.3.0:
   version "4.17.4"
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.4.tgz#78203a4d1c328ae1d86dca6460e369b57f4055ae"
 
+lodash@^4.17.4:
+  version "4.17.11"
+  resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.17.11.tgz#b39ea6229ef607ecd89e2c8df12536891cac9b8d"
+  integrity sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg==
+
```

Comment:
2 compatible versions of lodash and js-yaml have been installed. Not a huge deal, but a `yarn install --force` in root may solve it?

Reply:
@benjie good eye! no diff when I run that :( I'm not sure why, as 
`lodash@^4.0.0, lodash@^4.1.0, lodash@^4.15.0, lodash@^4.2.0, lodash@^4.2.1, lodash@^4.3.0`
and `lodash@^4.17.4` should all resolve `version "4.17.11"` according to [npm-semver](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004). Hopefully tree-shaking will fix this, as this is in `graphql-codemirror`

Reply:
Turns out Atlassian have written a `yarn-deduplicate` module that solves this; super handy! This is going to save me a tonne of work in future.

Anyway, I've pushed a commit that dedupes.

Reply:
(To your branch)

Reply:
thanks! that does help. we just need npm credentials to be able to release this now. i think it's good to merge now